.ci/install_cc_runtime.sh

```diff
@@ -0,0 +1,61 @@
+# Copyright (c) 2017 Intel Corporation
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+#!/bin/bash
+
+set -e
+
+echo "Retrieve cc-runtime repository"
+go get -d github.com/clearcontainers/runtime
+pushd $GOPATH/src/github.com/clearcontainers/runtime
+
+echo "Test cc-runtime"
+make check
+
+echo "Build cc-runtime"
+make
+
+echo "Install cc-runtime"
+sudo make install
+
+echo -e "Add cc-runtime as a new/default Docker runtime. Docker version \"$(docker --version)\" could change according to Semaphore CI updates."
+sudo mkdir -p /etc/default
+cat << EOF | sudo tee /etc/default/docker
+DOCKER_OPTS="-D --add-runtime cor=/usr/local/bin/cc-runtime --default-runtime=cor"
+EOF
+
+echo "Restart docker service"
+sudo service docker stop
+sudo service docker start
+
+echo "Install cc-runtime configuration.toml to /etc/clear-containers"
+sudo mkdir -p /etc/clear-containers
+sudo cp config/configuration_with_logs.toml /etc/clear-containers/configuration.toml
```

Comment:
Thanks for renaming this config file. But on reflection, I think it would be safer to only have a single config file and modify it as needed. That will make maintenance easier and minimise the chance of "code rot" if we add new options to the main config file but forget to update the "special" one for testing).

For example, I do this:

```
sed 's/^#\(\[runtime\]\|global_log_path =\)/\1/g' config/configuration.toml | sudo tee /etc/clear-containers/configuration.toml
```